### DISCUSSION TITLE
Darling Progress Report Q2 2019 | Darling Development Blog

### DISCUSSION THREAD

**Discussion by giscus** [darlinghq/darling#1390] [Blog comments]
# Darling Progress Report Q2 2019 | Darling Development Blog

We are very excited to say that in Q2 2019 (April 1 to June 30) we saw more community involvement than ever before. Many pull requests were submitted that spanned from bug fixes for our low level assembly to higher level modules such as the AppKit framework. Thanks to everyone for your contributions and we hope for this level of engagement to continue.

https://blog.darlinghq.org/2019/07/darling-progress-report-q2-2019.html?sc=1684935230483

**Comment by acheong08** [comment]
it's 2023

**Reply by CuriousTommy** [reply]
For context, we recently migrated to a new blog site. @facekapow does plan on releasing a new blog post, but he's currently busy with some IRL stuff.

**Reply by facekapow** [reply]
Yeah, I recently moved and I'm still fixing up a few things and organizing my stuff. I plan to have that new blog post done by the end of this month, though.

**Comment by DsoTsin** [comment]
Pareallel Universe?

**Comment by Naville** [comment]
Ah 2019, time to spend my life savings on BitCoin.

**Comment by acheong08** [comment]
Speaking of development, Darling has been pretty inactive for a bit over a month now. Anyone got any idea what is going on?

**Reply by CuriousTommy** [reply]
> Anyone got any idea what is going on?

It's still being worked on. It's just sometimes real life/work gets in the way, or we're currently busy working on other projects.

**Reply by rasikananda** [reply]
In order to make the time available for full-time development, how many people need how much money for how long to get it able to fully run the Adobe Creative Cloud suite?

**Comment by 678999776** [comment]
Very nice article, I enjoyed reading your post, very nice share, I want to twit this to my followers. Thanks!.
<a href="https://realactivator.com/asc-timetables-crack-registration-code-free/">aSc TimeTables crack</a>

**Comment by leadcafe** [comment]
<a href="http://bptp.support/">Best BPTP Projects in Faridabad</a> BPTP projects real estate in Faridabad... Find a collection of affordable residential projects in Faridabad from reputable builders

**Comment by crackzsoftware** [comment]
"Welcome to the world of data recovery with EaseUS! In this introduction, we'll explore how <a href="https://crackzsoftware.com/easeus-data-recovery-wizard-full-crack/">EaseUS Data Recovery crack</a> can help you retrieve your precious files."
